### PR TITLE
Separated QC Status

### PR DESCRIPTION
**Pull request by IkenoXamos**
Separated QC Status into Technical Status and Soft Skill Status. Previous references to QC Status now refer to Technical Status. May need further refactoring, as all logic regarding reports and averages are factoring in only Technical Status.

### CODE REVIEW

**Review by Quinn-Donnelly — CHANGES_REQUESTED** [review]
Due to the service level changes from PR #147 there are some changes that will impact functionality of this PR. Once reviewed please re request a review

File: src/app/quality-audit/components/quality-audit-list/quality-audit-list.component.ts [hunk 1]
```diff
@@ -109,21 +109,13 @@ export class QualityAuditListComponent implements OnInit, OnChanges {
     this.qcBatchNote.technicalStatus = qcNote.technicalStatus;
     this.qcBatchNote.softSkillStatus = qcNote.softSkillStatus;
     this.lastQcStatus = qcNote.technicalStatus;
-    // this.qaService.upsertQcBatchNote(this.qcBatchNote).toPromise().then(
```
Comment (RevatureGentry) [inline src/app/quality-audit/components/quality-audit-list/quality-audit-list.component.ts]: Do we need to be making this network call?

File: src/app/quality-audit/components/quality-audit-list/quality-audit-list.component.ts [hunk 2]
```diff
@@ -97,16 +98,18 @@ export class QualityAuditListComponent implements OnInit, OnChanges {
...
     }
   }
 
   handleQcBatchNoteChange(qcNote: QcNote) {
-    this.qcBatchNote.qcStatus = qcNote.qcStatus;
-    this.lastQcStatus = qcNote.qcStatus;
+    this.qcBatchNote.technicalStatus = qcNote.technicalStatus;
+    this.qcBatchNote.softSkillStatus = qcNote.softSkillStatus;
+    this.lastQcStatus = qcNote.technicalStatus;
     // this.qaService.upsertQcBatchNote(this.qcBatchNote).toPromise().then(
```
Comment (RevatureGentry) [inline src/app/quality-audit/components/quality-audit-list/quality-audit-list.component.ts]: Does this need to be uncommented?

**Review by RevatureGentry — CHANGES_REQUESTED** [review]
Double check that one network call being commented out. 

Can we persist both technical and soft skills QcStatus?

**Comment by RevatureGentry** [comment]
Great work!

**Pull request merged**